symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -58,4 +60,71 @@ protected function configureRoutes(RouteCollectionBuilder $routes)
         $routes->import($confDir.'/{routes}/'.$this->environment.'/**/*'.self::CONFIG_EXTS, '/', 'glob');
         $routes->import($confDir.'/{routes}'.self::CONFIG_EXTS, '/', 'glob');
     }
+
+    public static function bootstrapCli(array &$argv)
+    {
+        // consume --env and --no-debug from the command line
+
+        for ($i = 0; $i < \count($argv) && '--' !== $v = $argv[$i]; ++$i) {
```

Comment:
It would be nice if it can be moved in `symfony/symfony` directly, it's the last "tricky" part that is generated.

Reply:
makes sense! suggested in https://github.com/symfony/symfony/pull/29126#issuecomment-437092754

Reply:
and done in https://github.com/symfony/symfony/pull/29154